dask/dataframe/core.py

```diff
@@ -919,6 +920,15 @@ def _bind_operator_method(cls, name, op):
         """ bind operator method like DataFrame.add to this class """
         raise NotImplementedError
 
+    def estimate_size(self, _p=16):
```

Comment:
Hrm, true.  

OK, stepping back a bit I think I think my desire was to associate this function somehow to the Pandas term of `nunique`.  I think that this leverages intuition that users have already and also helps them to find this functionality.  I don't know the best way to make this association though.

We could add keywords and ignore them or err if `approximate=False`.  We could add an `nunique_approx(...)` method (though this seems ugly).  We could add a new function name as is done here.  We could do something more clever?